swan-cern/values.yaml

```diff
@@ -258,154 +258,61 @@ hadoopTokenGenerator:
   image: gitlab-registry.cern.ch/swan/docker-images/hadoop-token-generator:v2.1.2
 
 fluentd:
-  containerRuntime: containerd
+  enabled: true
   plugins:
     - fluent-plugin-rewrite-tag-filter
     - fluent-plugin-out-http
     - fluent-plugin-grok-parser
+    - fluent-plugin-route
+  tolerations:
+  - key: null
+    operator: Exists
+    effect: "NoSchedule"
```

Comment:
iiuc this is to allow the deployment of the fluentd daemonset on the master node. I wonder if we strictly need this. Currently, as this PR is now, we are discarding everything that comes from the kube-system namespace -- the master node only contains control plane pods of this namespace. Only if we decided to push logs from kube-system it would make sense to have this setting.

Reply:
After reviewing it, it is not needed due to what you said - we are discarding everything that comes from the kube-system namespace already.